backend/backend.py

```diff
@@ -72,6 +91,13 @@ def stop_video(self) -> None:
         """ Stops displaying the video feed and releases any resources
             captured. """
 
+    def send_camera_error(self, wait_for_publish: bool = False) -> None:
+        """ Sends the most recent exception for camera to the MQTT errors topic"""
+        message = { "trace": format_exc() }
+        self.publish_errors_func(message, wait_for_publish)
+        print(format_exc())
+        # TODO: Replace format_exc() with a logger
```

Comment:
This function does the exact same thing as you do in the `except` clause in `overlay.py` - maybe this code should just be moved inside `publish_errors` in `overlay.py`?